packages/shorebird_cli/lib/src/validators/validators.dart

```diff
@@ -65,6 +65,9 @@ class ValidationIssue {
 /// Checks for a specific issue with either the Shorebird installation or the
 /// current Shorebird project.
 abstract class Validator {
+  /// The short name of this validator.
+  String get name => '$runtimeType';
```

Comment:
minor but this probably won't be a human-readable name in a snapshot so do we want to give this a different name like `id` or something to indicate that you shouldn't use this in logs/stdout?

Reply:
Done.